### DISCUSSION TITLE
在login里能控制登录人数吗

### DISCUSSION THREAD

**Discussion by thinbug** [kbengine/kbengine#1509] [General]
在login里能控制登录人数吗？
在base里我可以计算出目前人数，但是在login/kbemain里，我如何能知道base里的人数？

尝试用KBEngine.globalData 是不行的，貌似只有base和cell里是同步的。

有没别的办法，我就想在线3000人时候不让登录了。

**Comment by Wandar** [comment]
可以让base写入文件,login读取

**Reply by Leovany** [reply]
如果这人数需要保证准确性，那么是要每时每刻去读取文件吗？ 还是定时去读取文件值？其次读写操作会不会耗性能？

PS : 我用的是笨方法，login和base 各创建个http，base有人数增减时候，http通知给login

**Reply by Wandar** [reply]
每隔十几秒读一次就行了,其实更好的办法是让玩家登陆进来,然后在Account里面做排队

**Comment by Leovany** [comment]
收到，谢谢~
